src/py_genome_track.cpp

```diff
@@ -216,7 +216,8 @@ GKPY_OMETHOD_BEGIN(GenomeTrackBuilder, set_data)
 		size_t stride = genome_track::dtype_size[dtype];
 		for (int dim = ndim; dim > 0; --dim) {
 			GK_CHECK((size_t)PyArray_STRIDE(py_data, dim - 1) == stride, value,
-					 "Data must have stride=1, consider making a copy with `np.array`");
+					 "Data detected having stride!=1, possibly because it is transposed. "
+					 "Consider making a copy with `np.array` to get non-strided data.");
```

Comment:
I'd remove the transpose, there's many ways to end up with a strided array and transpose isn't even the most common case